tests/integration/test_model.py

```diff
@@ -1024,3 +1046,130 @@ def test_predict_with_nullable_types(c):
         result,
         check_dtype=False,
     )
+
+
+# TODO - many ML tests fail on clusters without sklearn - can we avoid this?
+@skip_if_external_scheduler
+@pytest.mark.skipif(
+    sys.version_info < (3, 9),
+    reason="Some newer sklearn classes are only available with Python version >= 3.9",
```

Comment:
We use sklearn = 1.0.0 instead of sklearn >= 1.0.0 for our Python 3.8 integration tests:
https://github.com/dask-contrib/dask-sql/blob/main/continuous_integration/environment-3.8-dev.yaml#L30